react-calculator/src/App.js

```diff
@@ -0,0 +1,13 @@
+import './App.css';
+import React, { Component } from 'react';
+import Calculator from './Calculator';
+
+export default class App extends Component {
+  render() {
+    return (
+      <div id="app">
+        <Calculator></Calculator>
+      </div>
```

Comment:
```suggestion
      <div id="app">
        <Calculator />
      </div>
```
<Calculator></Calculator>를 html처럼 닫는 태그로 바로 닫아도 무방해요~
컴포넌트 내부에 데이터(children)이 없다면 바로 닫아줍시다!